src/Tribe/Admin/Event_Settings.php

```diff
@@ -1,13 +1,16 @@
 <?php
+namespace Tribe\Events\Admin;
 
 /**
  * Manages the admin settings UI in relation to events configuration.
  */
+use Tribe__App_Shop;
+use Tribe__Settings;
+use Tribe__Settings_Tab;
 use Tribe__Events__Main as Plugin;
 use Tribe\Admin\Troubleshooting as Troubleshooting;
 
-
-class Tribe__Events__Admin__Event_Settings {
+class Event_Settings {
```

Comment:
I think we're trying to use the plural for wherever we use Events or Tickets, right? Maybe setting this one as `Events_Settings` will make more sense.

Reply:
_Event_, in this case, acts as an adjective with _Settings_ being the noun that should be pluralized.

Reply:
Ok, gotcha. Although, with that naming, as a dev that comes at this for the first time I'd interpret that it's related to settings for a particular event rather than the "events" settings. Just a thought.

Reply:
Interesting perspective. I can't bring myself to go with `Events_Settings` because it isn't an English convention and my brain breaks. That being said, I get your point, so I propose `Tribe/Admin/Events/Settings.php` since the directories are intended to be nouns as well.